src/content.js

```diff
@@ -76,6 +79,7 @@ const domains = {
       { apply: url => url.replace(/\/(.*)-docs\//, '$1-docs-pr/') },
 
       { apply: url => commonRules.addDescription(url) },
+      { apply: (url, issueURL) => commonRules.addIssueURL(url , issueURL) },
```

Comment:
Since you're calculating the issue URL in the function, you don't need to pass it in as an argument.

```suggestion
      { apply: url => commonRules.addIssueURL(url) },
```